### PR TITLE
Update package-lock.json

### PR DESCRIPTION
**Pull request by T0TProduction**
@MeerBiene i just did `npm i` on my local machine, and it completely reset the package lock to version 1
should i commit this, and we kind of just accept the weirdness? 
it was done using node v12.16.1, npm 6.13.4

or do we just keep v2
to me it looks like it just added more specific info, which apparently is no issue for npm i , so we could also leave it be...

### CODE REVIEW

**Comment by MeerBiene** [comment]
In that case I think it doesnt really matter if the installation worked. 

What I can conclude is that I really should use an LTS Version for development. If we add a .nvmrc file, does that affect the lib when its installed, so that it would only run on the specified version or can we just npmignore it? Locking every developer in on the same version should probably prevent those kind of issues in the future.

What I also recognized, npm v>=7 achknowledged the yarn.lock file whereas npm 6 just ignored it. Would that mean someone using yarn would now have problems?

**Comment by MattIPv4** [comment]
It seems like there's a real mix of development tools being used here -- if NPM is being used for dev, why is there a yarn.lock file at all?

Imo, an `.nvmrc` file should be added that pins the project to Node.js LTS (14.x), and the lockfile updated to be correct for that (and yarn.lock removed?)

**Comment by MattIPv4** [comment]
Also, worth noting, every should always use `npm ci` to install deps if a lockfile is present, as it will respect the lockfile instead of making modifications to it.

**Comment by T0TProduction** [comment]
> It seems like there's a real mix of development tools being used here -- if NPM is being used for dev, why is there a yarn.lock file at all?
> 
> Imo, an `.nvmrc` file should be added that pins the project to Node.js LTS (14.x), and the lockfile updated to be correct for that (and yarn.lock removed?)

Sounds good IMO. 
Just for clarification though, If someone doesn't use nvm, the node version lock won't do anything, will it? Not a problem, just want to be aware of that potentially happening to contributors.

**Comment by MattIPv4** [comment]
Yeah, they will need to be using nvm -- I'd add a development section to the readme maybe the recommends installing nvm and making sure they've run `nvm use` && `npm ci` before working on the project.

**Comment by T0TProduction** [comment]
@MattIPv4 does the .nvmr look correct to you? i didn't find conclusive info online as to how to define a version that's not a single specific one
also, any changes to how i described the dev flow in the readme. didn't really spend too much time on it tbh, but it should have all the necessary info

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+v14.x
```
Comment (MattIPv4) [inline .nvmrc:1]:
```suggestion
v14.16.1
```
You'll want to specify an exact version (I think you can also use release aliases in theory, but having an exact version makes a dev environment more consistent and reproducible, same reason as following the lockfile for deps)

Comment (T0TProduction) [reply .nvmrc]: fair point, ill do just that.

**Review by MattIPv4 — APPROVED** [review]
seems sane

**Pull request merged**